### PR TITLE
Ra dspdc 479 datasetspecificv2f

### PR DESCRIPTION
**Pull request by raaidbroad**
Add necessary pieces for the Dataset Specific Analysis for V2F. This adds the necessary constants data, creates a new pipeline, creates source-of-truth integration test files, and adds an integration test.

### CODE REVIEW

File: v2f/src/it/scala/org/broadinstitute/monster/etl/v2f/V2FIntegrationSpec.scala [hunk 1]
```diff
@@ -80,7 +86,8 @@ class V2FIntegrationSpec extends PipelineSpec with Matchers with BeforeAndAfterA
       "variant-effect/transcript-consequences/*.json",
       "have written the correct transcript-consequences data"
     ),
-    ("variants/*.json", "have written the correct variants data")
+    ("variants/*.json", "have written the correct variants data"),
+    ("dataset-specific/*/part-*", "have written the correct dataset specific data")
```
Comment (danxmoran) [inline v2f/src/it/scala/org/broadinstitute/monster/etl/v2f/V2FIntegrationSpec.scala]: Did this glob pattern turn out to actually work?

File: v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/ExtractionPipeline.scala [hunk 2]
```diff
@@ -193,18 +193,18 @@ object ExtractionPipeline {
...
     *  Write all the converted and transformed Msg Objects to disk.
     *
-    * @param msgAndFilePaths the collection of Msg Objects and associated file paths that will be saved as a JSON file
+    * @param Messages the collection of Msg Objects and associated file paths that will be saved as a JSON file
     * @param filePath File pattern matching TSVs to process within the V2F analysis directory
     * @param outputDir the root outputs directory where the JSON file(s) will be saved
     */
   def writeToDisk(
-    msgAndFilePaths: SCollection[Msg],
+    Messages: SCollection[Msg],
```
Comment (danxmoran) [inline v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/ExtractionPipeline.scala]:
```suggestion
    messages: SCollection[Msg],
```

File: v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/ExtractionPipeline.scala [hunk 3]
```diff
@@ -193,18 +193,18 @@ object ExtractionPipeline {
...
     description: String,
     filePath: String,
     outputDir: String
   ): Unit = {
     MsgIO.writeJsonLists(
-      msgAndFilePaths,
+      Messages,
```
Comment (danxmoran) [inline v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/ExtractionPipeline.scala]:
```suggestion
      messages,
```

File: v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FConstants.scala [hunk 4]
```diff
@@ -270,3 +270,20 @@ case object VariantEffectTranscriptConsequences extends V2FConstants {
     "id" -> "variant_id"
   )
 }
+
+case object DatasetSpecificAnalysis extends V2FConstants {
+  override def tableName = "Dataset Specific Analysis"
+
+  override def filePath: String = "dataset-specific"
+
+  override def fieldsToConvertToMsgBoolean: Set[String] = Set("multi_allelic")
```
Comment (danxmoran) [inline v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FConstants.scala]: Isn't it already a boolean?

File: v2f/src/it/scala/org/broadinstitute/monster/etl/v2f/V2FIntegrationSpec.scala (hunk 1 above)
Comment (raaidbroad) [reply v2f/src/it/scala/org/broadinstitute/monster/etl/v2f/V2FIntegrationSpec.scala]: Whoa whattt. Maybe the globbing function used here is different? 😨

File: v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FConstants.scala (hunk 4 above)
Comment (raaidbroad) [reply v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FConstants.scala]: Yes, good catch, forgot to delete it

**Pull request merged**